README.md

```diff
@@ -61,6 +61,8 @@ yarn storybook
 
 Storybook should now be operational. Experiment with `EmbeddedChat` and its components, observing real-time changes in Storybook.
 
+Make sure the "Enable CORS" option is ON in your Rocket.Chat server. It can be found in Administration > Settings > General > REST API > Enable CORS. 
```

Comment:
Note : "Your content" Like that

Reply:
Sorry I do not understand, can you please elaborate?

Reply:
Note: Please make sure the "Enable CORS" option is ON in your Rocket.Chat server. It can be found in Administration > Settings > General > REST API > Enable CORS. It has to enabled to access the functionality of the app.